EasyPost/Exceptions/API/_Base.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
```

Comment:
Question: why does this file start with a underscore

Reply:
Just so it's at the top of the file browser since it's a base type that shouldn't be directly accessed.